h2o-py/tests/testdir_algos/modelselection/pyunit_PUBDEV_8675_modelselection_fail.py

```diff
@@ -42,20 +42,20 @@ def test_megan_failure():
         
         # assert z-values removed has smallest magnitude
         x = best_predictor_subset[ind]
-        x = [y for y in x if not (y in redundantPredictors)]
```

Comment:
I see that the test is passing now. I assume that the removal of this line is the key change, isn't it?

Reply:
Yes